scripts/json_device_merge.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python3
+
+import json
+import glob
+import os
+import hashlib
+
+bin_dir = os.path.join(os.environ["TOPDIR"], "bin")
+
+dev_overview = {"metdata_version": 1, "devices": {}}
```

Comment:
metadata_version?

Reply:
Yea, lynxis suggested it as we may change the format in some distant future.

Reply:
Fixed